configs/platforms/sles-11-s390x.rb

```diff
@@ -0,0 +1,12 @@
+platform "sles-11-s390x" do |plat|
```

Comment:
I followed the example I used for huaweios and the comment is just above the line that sets vmpooler_template.